Editor/Tasks/AbstractProcessTask.cs

```diff
@@ -30,40 +42,42 @@ public void Execute(Action onComplete = null)
 
 		protected void RunProcessWithArgs(string fileName, string args)
 		{
-			Debug.Log("Executing:: " + args);
-
-			var process = new Process();
+			Debug.Log("Executing:: " + args + "\r\nFrom " + WorkingDirectory);
```

Comment:
Minor: is the double colon intentional?

Reply:
Yeah but I can change it if you think it needs it